### DISCUSSION TITLE
Get waypoint of player

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2300] [Q&A]
How to get a waypoint of a player?
When entering a player's vehicle, their waypoint is set for you. This means that you can somehow obtain a player's waypoint.

Now how can i get a player's waypoint in a lua script?

**Comment by Yimura** [comment]
If there’s no native then it isn’t possible. I’m aware of the waypoint data existing in a certain data node but there’s no way to retrieve this from Lua.

**Reply by Yimura** [reply]
See the following class for more information: https://github.com/Yimura/GTAV-Classes/blob/master/netsync/nodes/player/CPlayerExtendedGameStateNode.hpp
